src/main/java/com/denizenscript/denizencore/utilities/CoreUtilities.java

```diff
@@ -709,6 +709,29 @@ public static String getClosestOption(List<String> strs, String opt) {
         return closest;
     }
 
+    public static int parseIndex(List list, String str) {
+        if (str.equalsIgnoreCase("last")) {

```

Comment:
Many Java default methods like `equalsIgnoreCase` are avoided in Denizen due to performance issues, note the use of a `CoreUtilities` alternative in the original code you edited.
In this case as multiple checks are made, just do `str = CoreUtilities.toLowerCase(str);` followed by `.equals` calls